examples/platform/silabs/wifi/icd/SleepManager.cpp

```diff
@@ -1,4 +1,6 @@
 #include "SleepManager.h"
+#include <lib/support/logging/CHIPLogging.h>
```

Comment:
I wonder if the sleep manager should follow the icd states

Reply:
For the SIT ICD use-case, there isn't much benefit to follow the ICD states other than the basic Fast / slow poll behavior.
Where we need to be synced is with the LIT ICD behavior to optimize the Wi-Fi connectivity based on the ICD operationnal states.

Reply:
Part of what i mean is that you are using a eventing based on commissioning events (open/close/completed) but really it is the same as following the kCommissioningWindowOpen KeepActiveFlagsValues. I suspect the wifi device will require to listen to not sleep and not filter mdns packets for any KeepActiveFlags